package.json

```diff
@@ -5,7 +5,7 @@
   "publisher": "Microsoft",
   "version": "0.1.1",
   "icon": "resources/catalog/CosmosDBExtension.png",
-  "aiKey": "add_key_here",
+  "aiKey": "AIF-37eefaf0-8022-4671-a3fb-64752724682e",
```

Comment:
Please also update to the latest version of the ads-extension-telemetry package (1.2.0) - there's some changes to the endpoint the data is sent to you'll want to get in (no functional changes on your side, just updating the package is enough)